k8s-scheduler-apiary.tf

```diff
@@ -57,7 +57,7 @@ resource "kubernetes_deployment" "beekeeper_scheduler_apiary" {
 
           liveness_probe {
             http_get {
-              path = "/actuator/health"
+              path = "/actuator/health/liveness"
```

Comment:
A comment here explaining why we doing this would be good.

Reply:
Documented it in the comment of the PR

Reply:
I meant as a comment in the code